package-lock.json

```diff
@@ -2284,7 +2284,7 @@
     },
     "axios": {
       "version": "0.15.3",
-      "resolved": "https://registry.npmjs.org/axios/-/axios-0.15.3.tgz",
+      "resolved": "http://registry.npmjs.org/axios/-/axios-0.15.3.tgz",
```

Comment:
You didn't modify `package.json` so there shouldn't be any changes to the lockfile. Can you omit this change from your PR?